controllers/admin.js

```diff
@@ -58,10 +70,14 @@ function getOAuthStrategies(aStored) {
 
 // Allow admins to set user roles and delete users
 exports.userAdmin = function (aReq, aRes, aNext) {
-  var options = nil();
+  //
+  var options = nil(); // NOTE: Inconsistent usage of object creation here
```

Comment:
@sizzlemctwizzle Cc: @Zren 

Is `nil` helper really needed here? So far I haven't seen any Object prototype lines in the controllers... but I haven't looked everywhere.